packages/strapi-plugin-upload/services/Upload.js

```diff
@@ -43,10 +43,9 @@ const combineFilters = params => {
 
 module.exports = {
   formatFileInfo({ filename, type, size }, fileInfo = {}, metas = {}) {
-    const ext = '.' + mime.extension(type) || path.extname(filename);
-    const baseName = path.basename(filename, path.extname(filename));
+    const ext = path.extname(filename);
 
-    const usedName = fileInfo.name || baseName;
+    const usedName = fileInfo.name || filename;
```

Comment:
Changing this to using filename will create unwanted hash. Can you update the call to `generateFileName`  function to call it with the previous `usedName` value hash stays clean

Reply:
Fixed